clang/runtime/dpct-rt/include/dpct/dpl_utils.hpp

```diff
@@ -12,15 +12,16 @@
 #define ONEDPL_USE_DPCPP_BACKEND 1
 #define __USE_DPCT 1
 
-#include <oneapi/dpl/execution>
 #include <oneapi/dpl/algorithm>
+#include <oneapi/dpl/execution>
 #include <oneapi/dpl/numeric>
 
-#include "dpl_extras/memory.h"
 #include "dpl_extras/algorithm.h"
-#include "dpl_extras/numeric.h"
+#include "dpl_extras/dpcpp_extensions.h"
 #include "dpl_extras/iterators.h"
+#include "dpl_extras/memory.h"
```

Comment:
It looks like except for the added `#include "group_utils.hpp"`, the other changes are just shifting the order of the includes. Does this matter or can we restore these unneeded changes?

Reply:
Yes this was caused by the format issue, let me revert to original.